src/ir/cost.h

```diff
@@ -51,15 +51,12 @@ struct CostAnalyzer : public Visitor<CostAnalyzer, Index> {
     return 2 + visit(curr->condition) + maybeVisit(curr->value);
   }
   Index visitCall(Call *curr) {
+    // XXX this does not take into account if the call is to an import, which
```

Comment:
The issue is that `->imported()` is on the Function object, not the Call (a Call no longer knows if its target is imported or not). We can get the Function from the module, but in general we allow optimizing functions before all other functions have been added. So if we did that here it would limit when cost.h can be used.

That might make sense, however, I think it's not clear we should be increasing the code for an import - yes, there is some extra possible overhead, but the real overhead is in the unseen code in the other function, which we can't estimate.